docs/tutorials/training.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Training
+---
+
+# Training
+There are learning algorithms present in the codebase for developers to get familiar with MOMA training. Various MOMA algorithms can be accessed [here](https://github.com/rradules/momaland/tree/main/momaland/learning).
```

Comment:
Can you update the link to point to the Farama github now? (Also make a quick search through the docs for links still pointing to Roxana's)

Reply:
Do you know what the URL will be for the documentation page? `momaland.farama.org`?

Reply:
It is the URL of the code, not the documentation :-)

Reply:
Yes, but in the `all-environments` table we are linking to the doc pages of those envs. Currently they are still `rradules.github.io/momaland/...`

Reply:
YOu can assume momaland.farama.org then